<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormImport</class>
 <widget class="QWidget" name="FormImport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnDoImport">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>570</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>23</y>
     <width>671</width>
     <height>541</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Select Source Type</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblFilePath">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>File Path</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblSQLServerIP">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Host Name/IP Address</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblMySQLPort">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Port</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblSQlServerAuth">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Authentication</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblSQLServerUserName">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>User Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblSQLServerPassword">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblSQLServerDatabase">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Database Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblWebAPIURL">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>URL</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblTargetTable">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Target Table</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblSourceTable">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Source Table</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QComboBox" name="cboFileType">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string>Excel file (2007 or later) (*.xlsx)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>CSV file (*.csv)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Microsoft SQL Server</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>MySQL</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>REST Api</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Oracle</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="txtImportPath">
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>9</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnImport">
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="txtSQLServerIP">
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>9</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSQLServerTest">
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Test Connection</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSQLServerSave">
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLineEdit" name="txtMySQLPort">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cboSQLServerAuth">
          <item>
           <property name="text">
            <string>SQL Server Authentication</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Windows Authentication</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtSQLServerUserName">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtSQLServerPassword">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtSQLServerDatabase">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtWebAPIURL">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cboTargetTable">
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Item</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Sales Transaction</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cboSourceTable"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="matchFieldsTableWidget">
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Target Field</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Source Field</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
